src/xPDO/Cache/xPDORethinkDBCache.php

```diff
@@ -0,0 +1,110 @@
+<?php
+/*
+ * This file is part of xPDO.
+ * 
+ * Copyright (c) Elizabeth Southwell <
```

Comment:
I followed the convention, https://github.com/modxcms/xpdo/blob/3.x/src/bootstrap.php#L5